.storybook/withThemeProvider.js

```diff
@@ -129,6 +129,7 @@ export const globalThemeProvider = {
     description: "Global theme for components",
     defaultValue: sageTheme.name,
     toolbar: {
+      title: "Theme",
       icon: "paintbrush",
       items: Object.keys(themes),
       showName: true,
```

Comment:
```suggestion
```

**suggestion** (non-blocking): while everything seems to work OK in your PR, including removing the warning, I'd still prefer to simply remove this `showName` property as that's what the warning is actually about. The `title: "Theme"` is ensuring the toolbar title still shows up correctly, and I don't see anything else wrong - although it's entirely possible I've missed something.